packages/combo-box/test/lazy-loading.test.js

```diff
@@ -166,7 +166,7 @@ describe('lazy loading', () => {
 
       // eslint-disable-next-line prefer-arrow-callback
       describe('when open', function () {
-        // eslint-disable-next-line no-invalid-this
+        // eslint-disable-next-line @typescript-eslint/no-invalid-this
```

Comment:
Consider removing `this.timeout(15000)` altogether - it's most likely a leftover from the old test runner.
Tests should be much faster nowadays. Also, you can use the arrow callback in `describe` above.

Reply:
Done.